qa/L0_perf_analyzer/test.sh

```diff
@@ -635,6 +635,27 @@ for PROTOCOL in grpc http; do
         set -e
     done
 
+    # Testing that trace logging works
+    set +e
+    TRACE_FILE="trace.json"
+    rm ${TRACE_FILE}*
+    $PERF_ANALYZER -v -i $PROTOCOL -m simple_savedmodel_sequence_object -p 2000 -t5 --sync --trace-file $TRACE_FILE \
+    --trace-level TIMESTAMPS --trace-rate 1000 --trace-count 100 --log-frequency 10 \
```

Comment:
Should we also test `--trace-level TENSORS `? Or do we expect the same output regardless of the two options?

Reply:
We could. [L0_cmdline_trace](https://github.com/triton-inference-server/server/blob/main/qa/L0_cmdline_trace/test.sh) already tests that the HTTP/GRPC clients trace correctly across different permutations, so I was trying to avoid repeating those tests. My plan with testing was to ensure tracing gets enabled correctly when the trace command line options are passed into perf_analyzer (which feeds them into the HTTP/GRPC clients), but I can add additional tests if you think they'd be useful.

Reply:
Looks like the test coverage is good. Thanks for pointing me to `L0_cmdline_trace`.